### DISCUSSION TITLE
Malware Detected - MXWTPTTTYKDUYVXJ.exe

### DISCUSSION THREAD

**Discussion by Ceekah1** [chaiNNer-org/chaiNNer#2253] [General]
malware running in background that was located in chaiNNer folder. Self duplicating process, have to force kill to remove.

**Comment by stonerl** [comment]
I did a quick [Google Search](https://www.google.com/search?q=MXWTPTTTYKDUYVXJ.exe). `MXWTPTTTYKDUYVXJ.exe` seems to be malware, but is not related to chaiNNer. It is most likely a coincidence that you found it in the chaiNNer folder.

**Reply by Ceekah1** [reply]
Yes, you're correct and Im not accusing anyone. 
To be completely fair Ive had chaiNNer on my pc for awhile and only recently it has popped up. So unless its dormant in Chainners files I would suspect its completely unrealated. 

-Blocked Website Details-
Malicious Website: 1
, C:\Users\XXXXXXXX\AppData\Roaming\chaiNNer\ffmpeg\MXWTPTTTYKDUYVXJ.exe, Blocked, -1, -1, 0.0.0, , 

-Website Data-
Category: RiskWare
Domain: de.monero.herominers.com
IP Address: 141.95.126.31
Port: 1112
Type: Outbound
File: C:\Users\XXXXXXXX\AppData\Roaming\chaiNNer\ffmpeg\MXWTPTTTYKDUYVXJ.exe
